README.md

```diff
@@ -29,7 +29,7 @@ databases are required.
 - **Data Management**: Handle and version large datasets, files, and models
   effectively right from VS Code.
 
-> **Note**: We are in public beta and welcome early adopter feedback!  
+> **Note**: We are in public beta and welcome early adopter feedback!
```

Comment:
Ugh why is Restyled trying to un-do my perfectly valid MD new line? 🙁

Reply:
I have not seen this until today.

Reply:
It is definitely not a required check, may have got turned on when we open-sourced the repository. Ignore it for now.

Reply:
Ignoring no worries, this stuff is not important of course! Thanks